### ISSUE TITLE
Actions from Private Repos are not retrieved/displayed

### ISSUE THREAD

**Issue by vikas027** [chriskinsman/github-action-dashboard#26]
Hello @chriskinsman ,

Similar to the [issue](https://github.com/chriskinsman/github-action-dashboard/issues/13) raised by @tgdfool2 , I am also unable to see any actions in the private repos. I have very recent workflow runs.

In the [comment](https://github.com/chriskinsman/github-action-dashboard/issues/13#issuecomment-938192890), you mentioned that we need to install _GitHub Report_ in order to get workflows in private repos in the dashboard.
Can you please elaborate a bit on what is GitHub report and how I install it?

> You need to make sure that you installed the GitHub report for the organization that owns the private repos.

Thanks.

**Comment by chriskinsman** [comment]
That was a typo.  I meant you have to install the GitHub application into the organization that owns the private repos.

We use this with private repos currently with now issues so I would wonder if it wasn't installed into the correct organization?

**Comment by vikas027** [comment]
> We use this with private repos currently with now issues so I would wonder if it wasn't installed into the correct organization?

No, @chriskinsman . I am sure I have installed into the correct organization. I was able to see the actions of the public repos, it is only private ones which are not working.

**Comment by JoshMcCullough** [comment]
FWIW, private repos are showing up fine for the instance I just stood up.
